DEV_SETUP.md

```diff
@@ -0,0 +1,50 @@
+# Getting set up
+
+To get started, you'll need VSCode and a local installation of Python 3.x. 
```

Comment:
is there a min version of Python 3.x? IIRC In our previous experiment porting SK to python we were using features from 3.11.